packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewHostApi.m

```diff
@@ -0,0 +1,176 @@
+                  loadRequest:(nonnull FWFNSUrlRequestData *)request
+                        error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  NSURLRequest *urlRequest = FWFNSURLRequestFromRequestData(request);
+  if (!urlRequest) {
+    *error = [FlutterError errorWithCode:@"CreateNSURLRequestFailure"
+                                 message:@"Failed instantiating an NSURLRequest."
+                                 details:[NSString stringWithFormat:@"Url was: '%@'", request.url]];
+    return;
+  }
+  [[self getWebViewInstance:instanceId] loadRequest:urlRequest];
+}
+
+- (void)webViewWithInstanceId:(nonnull NSNumber *)instanceId
+           setCustomUserAgent:(nullable NSString *)userAgent
+                        error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  [[self getWebViewInstance:instanceId] setCustomUserAgent:userAgent];
+}
+
+- (nullable NSNumber *)
+    webViewWithInstanceIdCanGoBack:(nonnull NSNumber *)instanceId
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return @([self getWebViewInstance:instanceId].canGoBack);
+}
+
+- (nullable NSString *)
+    urlForWebViewWithInstanceId:(nonnull NSNumber *)instanceId
+                          error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  return [self getWebViewInstance:instanceId].URL.absoluteString;
+}
+
+- (nullable NSNumber *)canGoForwardInstanceId:(nonnull NSNumber *)instanceId
+                                        error:(FlutterError *_Nullable __autoreleasing *_Nonnull)
+                                                  error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)createInstanceId:(nonnull NSNumber *)instanceId
+    configurationInstanceId:(nonnull NSNumber *)configurationInstanceId
+                      error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (void)evaluateJavaScriptInstanceId:(nonnull NSNumber *)instanceId
+                    javascriptString:(nonnull NSString *)javascriptString
+                          completion:(nonnull void (^)(NSString *_Nullable,
+                                                       FlutterError *_Nullable))completion {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSNumber *)
+    getEstimatedProgressInstanceId:(nonnull NSNumber *)instanceId
+                             error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  // TODO(bparrishMines): Implement
+  @throw [NSException exceptionWithName:@"UnsupportedException" reason:nil userInfo:nil];
+}
+
+- (nullable NSString *)getTitleInstanceId:(nonnull NSNumber *)instanceId
```

Comment:
`getTitleForWebViewWithIdentifier:...`

Reply:
I went with `titleForWebViewWithIdentifier` instead. It looks like `get` is reserved  for retrieving values indirectly: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html